### PR TITLE
Follow redirects

### PR DESCRIPTION
**Pull request by manuel-di-iorio**
By default, rss-parser will now follows a max of 1 redirect. It is configurable with the new *optional* `settings` argument of the method `parseURL()`, via the `maxRedirects` property (when it is 0, redirects are not followed)

PS: sorry for the commit called 'browserify'. I later realized from your readme that it was a placeholder name 😅

### CODE REVIEW

**Comment by bobby-brennan** [comment]
Looks like there's something funny going on with character encodings in the test files as well. What versions of `entities` and `xml2js` do you have installed?

**Comment by manuel-di-iorio** [comment]
entities 1.1.1
xml2js 0.4.17

I'm on windows if care

**Comment by bobby-brennan** [comment]
Hmm windows might be the culprit. Could also be xml2js 0.4.15 vs 0.4.17.

Can you actually just revert `test/`, `dist/`, and `package.json`? I'll cut a minor release once this is merged.

Thanks for putting this together!

**Comment by manuel-di-iorio** [comment]
Done! :+1:

File: index.js [hunk 1]
```diff
@@ -190,7 +190,15 @@ Parser.parseString = function(xml, callback) {
   });
 }
 
-Parser.parseURL = function(feedUrl, callback) {
+Parser.parseURL = function(feedUrl, settings, callback) {
+  if (!callback) {
+    callback = settings;
+    settings = {
```
Comment (manuel-di-iorio) [inline index.js]: Correct!

**Comment by bobby-brennan** [comment]
Thanks!

**Pull request merged**